README.md

```diff
@@ -88,6 +152,8 @@ jason: Lose
 - [x] display results
 
 ## Considerations
-Q1: How should we name players inside the Players class? </br>
-Q2: Should we use multiple decks? </br>
-Q3: State in Controller (even if private) is bad practice?</br>
\ No newline at end of file
+- Q: Would it make sense to directly call draw twice in the init block when creating an Init object?
+- Q: Why do I not pass the whole deck as parameter to draw()?
+- Q: Is it not costly to always return new instances of State?
+- Q: Mutable handcards vs immutable handcards -> adding a new card always returns a new instance of HandCards
```

Comment:
I'd recommend using immutable `HandCards`, since it's a practice project.
It's not a large-scale system, so returning a new object each time shouldn't be a big concern.